front/components/layout/Links.vue

```diff
@@ -90,6 +91,13 @@ export default {
 
       return pageOn
     }
+  },
+  methods: {
+    async exploreClicked() {
+      const ideas = await getAllIdeas(this.$amplifyApi)
+      const firstIdea = ideas.ideas[0]
+      this.$router.push(`/i/${firstIdea.shortId}/${firstIdea.slug}`)
```

Comment:
@f-prime this seems a bit fragile -- is this how we are generally implementing path changes? is there some sort of urlresolver?

(totally aware that i'm a bit out of the loop, so just looking for feedback on whether this is our preferred method -- if it is, that's fine)

Reply:
Yeah thanks for pointing that out, you are right. This case is a little bit weird because what I am trying to do is drop the user directly into the Idea page which already has the swiping and treat that as the explore page. Are these pages not the same? Is Explore different from, say, https://beta.dailyidea.com/i/ygik8t/test ? The design seems to suggest that they are the same.

Regardless, I should be at least checking if the getAllIdeas method succeeds so I will fix that because I agree it is fragile. The question is now, what happens if getAllIdeas fails?

Reply:
Let's let that be a problem when it's a problem.  I'm going to for now just assume that getAllIdeas method can't / doesn't fail (obviously not true).

For now I'll just document this as a potential future tech debit / refactor issue and I'm going to merge this.